pages/api/graphql.ts

```diff
@@ -0,0 +1,25 @@
+import { ApolloServer, gql } from 'apollo-server-micro'
+
+const typeDefs = gql`
+  type Query {
+    sayHello: String
+  }
+`
+
+const resolvers = {
+  Query: {
+    sayHello() {
+      return 'Hello World!'
+    },
+  },
+}
+
+const apolloServer = new ApolloServer({ typeDefs, resolvers })
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+}
+
+export default apolloServer.createHandler({ path: '/api/graphql' })
```

Comment:
this is super nifty! 😎 
https://www.apollographql.com/docs/apollo-server/v1/servers/micro/

Reply:
Yeah it is pretty cool. The next integration makes it hot-reloadable